website/server/models/user/methods.js

```diff
@@ -20,10 +20,11 @@ schema.methods.getGroups = function getUserGroups () {
   return userGroups;
 };
 
-schema.methods.sendMessage = async function sendMessage (userToReceiveMessage, message) {
+schema.methods.sendMessage = async function sendMessage (userToReceiveMessage, translatedMessage, message) {
```

Comment:
Ok. About the default value: || would catch anything that resolves to false, not just undefined. It's unlikely to get passed, but isn't it better to be on the safe side?